server/app/routes/index.js

```diff
@@ -23,6 +23,7 @@ router.use('/categories', require('./category'));
 
 // Add API specific middleware
 router.use(require('../middlewares/auth-guard'));
+router.use(am(require('../middlewares/perm')));
```

Comment:
I think you could merge auth-guard and perm file, as one does not come without the other...

Reply:
Yes!